cob_bringup/drivers/canopen_402.launch

```diff
@@ -19,6 +20,6 @@
 	<node ns="$(arg component_name)" name="$(arg component_name)_joint_state_controller_spawner" pkg="controller_manager" type="spawner" args="joint_state_controller" respawn="true" />  
   
 	<!-- control_mode_adapter -->
-	<node ns="$(arg component_name)" name="control_mode_adapter" pkg="cob_control_mode_adapter" type="cob_control_mode_adapter_node" cwd="node" respawn="false" output="screen"/>
+	<node if="$(arg mode_adapter)" ns="$(arg component_name)" name="control_mode_adapter" pkg="cob_control_mode_adapter" type="cob_control_mode_adapter_node" cwd="node" respawn="false" output="screen"/>
```

Comment:
I thought we agreed on moving the control_mode_adapter out of the driver launch file...